index.html

```diff
@@ -3989,10 +3989,25 @@ <h3>JSON-LD</h3>
 
         <p>
 As elaborated upon in Section <a href="#json-processing"></a>, some software
-applications might not perform full JSON-LD processing.
-<a>Conforming document</a> authors are urged to not use JSON-LD keywords that
-affect document processing in the `@context` value, such as globally setting
-`@base` and `@language`, in order to increase interoperability.
+applications might not perform full JSON-LD processing. In order to increase
+interoperability, <a>conforming document</a> authors are urged to not use
+JSON-LD features that are not easily detected when not performing full
+JSON-LD processing. This guidance includes:
+        </p>
+
+        <ul>
+          <li>
+Not using JSON-LD keywords in the `@context` value that modify document
+processing, such as globally setting `@base` and `@language`, in order to
+increase interoperability.
```

Comment:
```suggestion
processing, such as globally setting `@base` and `@language`.
```
This has been stated already in the paragraph before the bullet list (and this actually applies to all items).

Reply:
See https://github.com/w3c/vc-data-model/pull/1276#discussion_r1325510657